### PR TITLE
Add tags to error error output

### PR DESCRIPTION
**Pull request by 0fuzz**

### CODE REVIEW

File: validator.go [hunk 1]
```diff
@@ -453,13 +454,13 @@ func (v *Validate) StructExcept(current interface{}, fields ...string) error {
...
 // You will need to assert the error if it's not nil i.e. err.(validator.ValidationErrors) to access the map of errors.
-func (v *Validate) Struct(current interface{}) error {
+func (v *Validate) Struct(current interface{}, errTagName string, baseTagName string) error {
 	v.initCheck()
 
 	errs := v.errsPool.Get().(ValidationErrors)
 	sv := reflect.ValueOf(current)
-
-	v.tranverseStruct(sv, sv, sv, blank, blank, errs, true, false, false, nil, false)
+	fmt.Println(errs)
```
Comment (gm42) [inline validator.go]: this one maybe needs to go?

**Comment by joeybloggs** [comment]
hello @0fuzz I'm a little confused about this one, was there some functionality that you desired that is missing; can't quite wrap my head around it?

**Comment by 0fuzz** [comment]
Hey joey, yes sorry we use it in some other way and I hit pull request per accident ;) But this is quite custom and I did close it. Sorry! ;)

**Comment by joeybloggs** [comment]
No worries, and if you'd ever like something added to the library don't hesitate to ask.

**Pull request closed without merging**